data/json/hobbies.json

```diff
@@ -1067,6 +1067,37 @@
     "proficiencies": [ "prof_athlete_basic", "prof_athlete_expert" ],
     "traits": [ "TOUGH", "FLEET", "PAINRESIST" ]
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "soccer_beginner",
+    "name": "Soccer (Beginner)",
+    "description": "You used to kick a ball around with your friends, having fun together. Though it may not be the case anymore.",
```

Comment:
```suggestion
    "description": "You used to kick a ball around with your friends, having fun together.  Though it may not be the case anymore.",
```